tests/openwisp2/sample_firmware_upgrader/migrations/0007_create_device_firmware.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.0.8 on 2020-07-07 17:00
```

Comment:
I am not sure whether this migration is really needed. From the code, I understand that `create_device_firmware_for_connections` will try to create a `DeviceFirmware` instance for existing devices. From the scope of sample app, I don' think it is necessary. 

@nemesisdesign what do you think about this?

Reply:
@pandafy you're correct, although it can be useful for developers wanting to extend fw-upgrader and looking for a reference implementation, so we can leave this data migration.
@R9295 why did you add this migration? Is it because you wanted the two apps to have the same number of migrations?
We can keep it for the reason expressed above.